add study lusc_cptac_2021

egarcialara opened the pull request:
# What?
- added new study [Shankha Satpathy et al. "A proteogenomic portrait of lung squamous cell carcinoma" Cell 2021](https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6#secsectitle0020)
https://github.com/cBioPortal/datahub/issues/1464

:information_source:  See [README](https://github.com/cBioPortal/datahub/blob/lusc_cptac_2021/public/lusc_cptac_2021/README.md) for transformation process

Cancer studies updated in this pull request:
- lusc_cptac_2021

# checks
For all pull requests:
- [x] Passes validation

For a new study (in addition to above):
- [x] Does study name and study ID follow our convention? e.g. Tumor_Type (Institue, Journal Year); brca_mskcc_2015
- [x] is study meta data complete? e.g. pmid, group of PUBLIC
- [x] were all samples profiled with WES/WGS? If not, is gene panel file curated?
- [x] are oncotree codes of all samples curated; Cancer Type and Cancer Type Detailed needs to be added in addition to Oncotree Code
- [x] clinical sample and patient data with meta files
- [x] mutations data with meta files
- [x] MAF is based on hg19
- [ ] ~~MAF with 2 isoforms: uniprot and mskcc~~ _Not applicable_
- [x] CNA data with meta files
- [ ] ~~CNA segment data with meta files~~ _Not applicable_
- [x] Expression data including z-scores with meta files
- [x] Case-lists for all profiles.
- [ ] Manual checking (Niki or JJ): Triage or private Portal link here

Testing portal instance:
https://triage.cbioportal.mskcc.org/study/summary?id=lusc_cptac_2021

egarcialara commented:
In this study, data_log2CNA.txt has many repeated values per sample.

SRodenburg commented on public/lusc_cptac_2021/meta_methylation_epic.txt:
```diff
@@ -0,0 +1,8 @@
+cancer_study_identifier: lusc_cptac_2021
+genetic_alteration_type: METHYLATION
+datatype: CONTINUOUS
+stable_id: methylation_epic
```
stable ID in methylation is a limited namespace. Validator throws an error:
```
ERROR: meta_methylation_epic.txt: Invalid stable id for genetic_alteration_type 'METHYLATION', data_type 'CONTINUOUS'; expected one of [methylation_hm27, methylation_hm450]; value encountered: 'methylation_epic'
```

egarcialara replied:
Hi,
I have kept methylation_epic as stable id, because that is the one that @yichaoS used in a previous study ([luad_cptac_2020](https://github.com/cBioPortal/datahub/pull/1503)), so I guessed that this stable id will pass validation in cBioPortal on their side?

I changed the stable id lusc_cptac_2021_custom_all to lusc_cptac_2021_all to avoid the second error message.

SRodenburg replied:
@yichaoS can you verify what Elena says here? It it OK to use this stable ID?

yichaoS replied:
@SRodenburg @egarcialara Yes, we created this new profile in portal and the stable ID is `methylation_epic`. (for methylations done on the EPIC platform). I've changed the methylation profile meta in other CPTAC studies to this as well.

SRodenburg replied:
OK I guess then the validator needs an update :)

yichaoS replied:
@SRodenburg I've updated the `allowed_data_type` sheet with the new data types in the cbioportal repo.

SRodenburg reviewed (changes requested):
at the moment the study does not validate - see also the CircleCI test...

```
ERROR: meta_methylation_epic.txt: Invalid stable id for genetic_alteration_type 'METHYLATION', data_type 'CONTINUOUS'; expected one of [methylation_hm27, methylation_hm450]; value encountered: 'methylation_epic'


ERROR: -: No case list found with stable_id 'lusc_cptac_2021_all', consider adding 'add_global_case_list: true' to the meta_study.txt file
```

yichaoS commented:
@egarcialara could you please add back the README for this PR as well? And what's the data source for this PR? I'm comparing the paper figures with oncoprint in testing portal and the percentage of variants seems off.

egarcialara commented:
Hi @yichaoS 

- I thought I had uploaded the README. Here it is now!
- The data source is mostly the Supplementary tables. I downloaded all data from the Supp. materials in the paper. I chose these tables instead of the ones from LinkedOmics as: mutations is then a MAF file, miRNA has the correct IDs format, and phospho/acetyl/ubiquityl proteomes have data at “gene-level”. Circular RNA was only present in LinkedOmics and I uploaded it now to this repo.
- There was also the little issue that dropped a few rows unnecessarily, so now the number of rows is closer to the original files :) If any row was dropped, it’s written down in the README. I checked, and NF1 has at least 5 nonsense mutation events in the updated table. 
- I removed CNA, log2CNA and the relevant meta/case list files.

If it makes your work easier (the extra 1 or 2 rows being comments / header not counted in study):

data | original | study
--- | --- | ---
mutations | 35530 | 35462
methylation | 16479 | 16478
rna | 21794 | 21792
mirna | 2440 | 2439
protein | 11114 | 11111
phospho | 7732 | 7730
acetyl | 2520 | 2518
ubiquityl | 3937 | 3935
cRNA (linkedomics) | 306 | 306

All together I updated the tables: mutations, methylation, expression (and zscores), miRNA (and zscores), and added circular RNA. I added meta and cases list file for cRNA, and re-did the cases lists for all.
